ale_linters/kotlin/ktlint.vim

```diff
@@ -0,0 +1,54 @@
+" Author: Francis Agyapong <francisagyapong2@gmail.com>
+" Description: Lint kotlin files using ktlint
+
+let g:ale_kotlin_ktlint_exec = get(g:, 'ale_kotlin_ktlint_exec', 'ktlint')
```

Comment:
Sorry I didn't notice this earlier, but this option should be named `executable` so it matches the other option names.

If you like, you can also use `ale#Set` to configure variables, if you hate typing the same name twice. Look for examples in the codebase.